variables.tf

```diff
@@ -2,19 +2,14 @@
 # Common
 ##########################################
 
-variable "tags" {
-  description = "Tags applied to your resources"
-  default     = {}
-}
-
-variable "region" {
-  description = "AWS deployment region"
+variable "resource_naming_prefix" {
```

Comment:
Allow users to specify a single prefix to be applied to all of their resources, with optional overrides applying to the model, endpoint and ecr_repo resources if they desire